roles/openshift_health_checker/openshift_checks/logging/logging.py

```diff
@@ -27,7 +30,7 @@ def is_first_master(self):
         return masters[0] == hostname
 
     def run(self):
-        pass
+        return {}
```

Comment:
I believe the more appropriate change would be to make this a `LoggingMixin` or such, and then this class wouldn't be listed as a check. For now, simply making it comply with the API and return a dict, avoiding an unhandled exception.